risc0/zkvm/src/host/server/exec/executor.rs

```diff
@@ -771,3 +776,58 @@ impl<'a> ExecutorImpl<'a> {
         ))
     }
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+struct ExecutorStats {
+    execution_time: Duration,
+    segment_count: usize,
+    total_prover_cycles: u64,
+    user_instruction_cycles: u64,
+    cycles_used_ratio: f64,
+}
```

Comment:
It doesn't seem like `ExecutorStats` is used anywhere else, so a struct seems like overkill when we could just have:

```
impl Session {
  fn log() {
    // compute values, print them here
    ...
}
```

Reply:
Makes sense, thanks for the input!